lib/stripe/payment_methods/card.ex

```diff
@@ -98,7 +100,10 @@ defmodule Stripe.Card do
   If you want to create a card with your server without a token, you
   can use the low-level API.
   """
-  @spec create(map, Keyword.t()) :: {:ok, t} | {:error, Stripe.Error.t()}
+  @spec create(params, Keyword.t()) :: {:ok, t} | {:error, Stripe.Error.t()}
+        when params: %{
+               optional(:customer) => Stripe.id() | Stripe.Account.t()
+             }
```

Comment:
@begedin I guess what I was wondering is `what` keys should be included here.  I see in other module that it's only a small subset.  Also does `clean up the params` refer to only this spot in the guard clause?  Or is there other spots this req is asking for?

Reply:
For example, this is only a limited list of the Subscription struct's keys...

https://github.com/code-corps/stripity_stripe/blob/2.0-beta/lib/stripe/subscriptions/subscription.ex#L74

Reply:
In the example of a subscription, if you take a look at the documentation for the create action

https://stripe.com/docs/api#create_subscription

you will see that the spec matches that list much better (though we're still missing `prorate` as an optional boolean).

So to conclude, each action's `@spec` should match the arguments list for that action in stripe's documentation.